generators/client/templates/react/_package.json

```diff
@@ -8,55 +8,58 @@
   ],
   "dependencies": {
     "availity-reactstrap-validation": "1.4.3",
-    "axios": "0.16.2",
-    "bootstrap": "4.0.0-beta",
+    "axios": "0.17.0",
+    "bootstrap": "4.0.0-beta.2",
     "loaders.css": "0.1.2",
     "lodash": "4.17.4",
-    "moment": "2.18.1",
-    "react": "15.6.2",
-    "react-addons-css-transition-group": "15.6.2",
-    "react-addons-transition-group": "15.6.2",
-    "react-dom": "15.6.2",
-    "react-jhipster": "0.1.1",
+    "moment": "2.19.1",
+    "react": "16.0.0",
+    "react-addons-css-transition-group": "16.0.0-alpha.3",
+    "react-addons-transition-group": "16.0.0-alpha.3",
+    "react-dom": "16.0.0",
+    "react-hot-loader": "3.1.1",
     "react-icons": "2.2.7",
-    "react-redux": "5.0.5",
-    "react-redux-loading-bar": "2.9.2",
-    "react-router-dom": "4.1.2",
-    "react-transition-group": "1.2.0",
-    "reactstrap": "5.0.0-alpha.1",
-    "redux": "3.7.1",
+    "react-jhipster": "0.1.1",
+    "react-redux": "5.0.6",
+    "react-redux-loading-bar": "2.9.3",
+    "react-router-dom": "4.2.2",
+    "react-transition-group": "2.2.1",
+    "react-translate-component": "0.14.0",
```

Comment:
I think this lib is not required as translation is done in the react-jhipster lib